### ISSUE TITLE
slides Per Column issue

### ISSUE THREAD

**Issue by Anas-shamia** [nolimits4web/swiper#3754]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: EXACT RELEASE VERSION OR COMMIT HASH, HERE.
* Platform/Target and Browser Versions: PLATFORM CLIENT YOU ARE TARGETING SUCH AS macOS, Windows, CORDOVA, IOS, ANDROID, CHROME, ETC.
* Live Link or JSFiddle/Codepen or website with isssue: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_.

### What you did
I tried to use slidesPerColumn with its css but its not working after new update.
please update Stackblitz code

### Expected Behavior
Slider with two columns

### Actual Behavior
All slides display under each other 

P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/swiper)
for that.

Before you open an issue, please check if a similar issue already exists or has been closed before.

**Comment by nolimits4web** [comment]
Your issue is closed because it is not filled correctly. Before you open an issue please review the [contributing guideline](https://github.com/nolimits4web/Swiper/blob/master/CONTRIBUTING.md).

Please fill the issue template correctly and:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with isssue

It will save a lot of time for other contributors to check your issue and fix it.

**Comment by AvdeevaEvgeniya** [comment]
The author wrote everything correctly, in the new version the slider for several columns does not work correctly, and you just close the problem instead of solving it!

**Comment by vhoyer** [comment]
No, @AvdeevaEvgeniya, the issue is really missing the required information, if you have the same problem and want it to be fixed as soon as possible, consider opening a new issue yourself with all required information: library version, platform, reproduction link and steps, and a clear non ambiguous description of whats happening and how to reproduce it.

Complain will solve nothing.

**Comment by vhoyer** [comment]
Anyway, "I" found a workaround to this, use `display: grid` on `.swiper-wrapper` instead of the default `flex`

```scss
  .swiper-wrapper {
      display: grid;
      grid-template-columns: 1fr;
      grid-template-rows: auto auto; // here is the amount of slidesPerColumn you want
      grid-auto-flow: column;

      & > .swiper-slide {
        height: fit-content;
      }
    }
```

**Comment by VitorTheGreat** [comment]
Hello everyone!

@vhoyer I used your solution and it worked successfully when used without the following configuration:
```
        centeredSlides: false,
        loop: false,
```

When I try to use with these tow options set to TRUE, the slider breaks and all the slides are gone. I tried several solutions with css only but I could not get it. I wonder if you have tried your solution like this.

By the way, sorry to go back again in the issue!

Thanks!
